application/src/main/java/de/wuespace/telestion/project/daedalus2/example/SystemTPublisher.java

```diff
@@ -14,7 +14,7 @@
 public class SystemTPublisher extends AbstractVerticle {
 	public static void main(String[] args) {
 		Vertx vertx = Vertx.vertx();
-		vertx.deployVerticle(new MongoDatabaseService("daedalus2", "d2Pool"));
+		//vertx.deployVerticle(new MongoDatabaseService("daedalus2", "d2Pool"));
```

Comment:
Just remove lines

Reply:
Done in https://github.com/wuespace/telestion-project-daedalus2/pull/123/commits/af0fe65a6b7b9542a91a829c2215bcfa4664a5c7.